packages/ui/src/components/Checkbox/index.tsx

```diff
@@ -183,8 +235,15 @@ const StyledActivityContainer = styled.div`
 
 type CheckboxProps = {
   error?: string | ReactNode
+  /**
+   * @deprecated Size prop is deprecated and will be removed in next majour update.
+   */
   size?: number
+  /**
+   * @deprecated Progress prop is deprecated and will be removed in next majour update.
+   */
   progress?: boolean
+  helper?: string | ReactNode
```

Comment:
```suggestion
  helper?: ReactNode
```

ReactNode includes string